driver/controller_test.go

```diff
@@ -181,14 +181,20 @@ func (*fakeTagsDriver) UntagResources(context.Context, string, *godo.UntagResour
 }
 
 func TestControllerExpandVolume(t *testing.T) {
+	default_volume := &godo.Volume{
+		ID:            "volume-id",
+		SizeGigaBytes: (defaultVolumeSizeInBytes / giB),
+	}
 	tcs := []struct {
-		name string
-		req  *csi.ControllerExpandVolumeRequest
-		resp *csi.ControllerExpandVolumeResponse
-		err  error
+		name        string
+		req         *csi.ControllerExpandVolumeRequest
+		resp        *csi.ControllerExpandVolumeResponse
+		err         error
+		godo_volume *godo.Volume
```

Comment:
I'd name this just `volume` since the type is embedded in the `*godo.Volume` type. This would also keep the diff in this PR slightly smaller.

Reply:
Sure, done.